microk8s-resources/actions/enable.gpu.sh

```diff
@@ -9,6 +9,27 @@ readonly SOCKET="$SNAP_COMMON/run/containerd.sock"
 
 echo "Enabling NVIDIA GPU"
 
+read -ra ARGUMENTS <<< "$1"
+
+if [[ "$ARGUMENTS" == "force-system-driver" ]]
```

Comment:
Do you think we should have a check to detect if we have an nvidia GPU?

Reply:
Do you think the `force` flags should ignore that?